lib/shelley/src/Cardano/Wallet/Shelley/Launch/Cluster.hs

```diff
@@ -427,27 +391,252 @@ cliRetry tr msg processConfig = do
+
+            -- In order to get a working stake pool we need to.
+            --
+            -- 1. Register a stake key for our pool.
+            -- 2. Register the stake pool
+            -- 3. Delegate funds to our pool's key.
+            --
+            -- We cheat a bit here by delegating to our stake address right away
+            -- in the transaction used to registered the stake key and the pool
+            -- itself.  Thus, in a single transaction, we end up with a
+            -- registered pool with some stake!
+
+            let certificates = catMaybes
+                    [ pure stakeCert
+                    , pure poolRegistrationCert
+                    , pure dlgCert
+                    , mPoolRetirementCert
+                    ]
+            (rawTx, faucetPrv) <- preparePoolRegistration
+                tr dir ownerPub certificates pledgeAmt
+            tx <- signTx tr dir rawTx [faucetPrv, ownerPrv, opPrv]
+            submitTx tr socket name tx
+        , metadataUrl = T.pack metadataURL
+        , recipe = pr
+        }
+
+defaultPoolConfigs :: [PoolRecipe]
+defaultPoolConfigs = zipWith (\i p -> p {index = i}) [1..]
     [ -- This pool should never retire:
-      PoolConfig {retirementEpoch = Nothing}
-      -- This pool should retire almost immediately:
-    , PoolConfig {retirementEpoch = Just 3}
-      -- This pool should retire, but not within the duration of a test run:
-    , PoolConfig {retirementEpoch = Just 100_000}
-      -- This pool should retire, but not within the duration of a test run:
-    , PoolConfig {retirementEpoch = Just 1_000_000}
+      PoolRecipe
+        { pledgeAmt = 200 * millionAda
+        , retirementEpoch = Nothing
+        , poolMetadata = Aeson.object
+              [ "name" .= Aeson.String "Genesis Pool A"
+              , "ticker" .= Aeson.String "GPA"
+              , "description" .= Aeson.Null
+              , "homepage" .= Aeson.String "https://iohk.io"
+              ]
+        , delisted = False
+        , index = undefined
+        }
+
+    , PoolRecipe
+        { pledgeAmt = 100 * millionAda
+        , retirementEpoch = Just 3
+        , poolMetadata = Aeson.object
+              [ "name" .= Aeson.String "Genesis Pool B"
+              , "ticker" .= Aeson.String "GPB"
+              , "description" .= Aeson.Null
+              , "homepage" .= Aeson.String "https://iohk.io"
+              ]
+        , delisted = False
+        , index = undefined
+        }
+
+    -- This pool should retire, but not within the duration of a test run:
+    , PoolRecipe
+        { pledgeAmt = 100 * millionAda
+        , retirementEpoch = Just 100_000
+        , poolMetadata = Aeson.object
+              [ "name" .= Aeson.String "Genesis Pool C"
+              , "ticker" .= Aeson.String "GPC"
+              , "description" .= Aeson.String "Lorem Ipsum Dolor Sit Amet."
+              , "homepage" .= Aeson.String "https://iohk.io"
+              ]
+        , delisted = True
```

Comment:
A note here would be good I think. What does `delisted` mean for this pool? Will we be able to interact with it during the test run?

Reply:
e2027d4c51bc90a481344b78a76a1f763f5071b1 - btw there is some integration test that checks that the wallet also lists delisted pools (but with a delisted flag, I think)